sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -111,9 +111,9 @@ def main():
         interface.shutdown()
         return
 
-    B = np.loadtxt(sys.argv[1])
-    desired_sigmas = np.loadtxt(sys.argv[2])
-    ids = np.loadtxt(sys.argv[3])
+    B = np.loadtxt('./formation/' + sys.argv[1])
```

Comment:
I will commit the change once I have edited the wiki with the example. Maybe after explaining the example I will come with more small changes.